internal/sip/sip_test.go

```diff
@@ -65,10 +53,14 @@ func TestNew(t *testing.T) {
 	)
 	bdAIPPath := bornDigitalAIP.Join("Born-Digital-AIP")
 
-	bornDigitalSIP := fs.NewDir(t, "",
-		fs.WithDir("content"),
-		fs.WithDir("header"),
-	)
+	bornDigitalSIPName := "SIP_20201201_someoffice_someref"
+	bornDigitalSIPTempDir := fs.NewDir(t, "",
+		fs.WithDir(bornDigitalSIPName,
+			fs.WithDir("content"),
+			fs.WithDir("header"),
+		),
+	).Path()
+	bornDigitalSIPPath := filepath.Join(bornDigitalSIPTempDir, bornDigitalSIPName)
 
```

Comment:
I think it would good to have a test case for a valid born digital SIP name with no reference (e.g. "SIP_20010106_someoffice" -- I'm not sure if the current RegEx will allow it.